### ISSUE TITLE
Delete 2 duplicate glossary entries in Weblate

### ISSUE THREAD

**Issue by laralem** [bryceco/GoMap#576]
I know I have the option "Tools > Delete String" but it doesn't work.

https://hosted.weblate.org/translate/go-map/glossary/en/?checksum=ed516d3f6d0b636d
https://hosted.weblate.org/translate/go-map/glossary/en/?checksum=d76900c35328739e

Could someone with administrator rights please delete them for me?

**Comment by bryceco** [comment]
I tried deleting them as well without success. I'll try later as a raw edit using git.

**Comment by laralem** [comment]
Thank you. Maybe it is a problem with Weblate. Today I can't edit in "Zen mode" in Weblate either.

**Comment by laralem** [comment]
Weblate is really crazy. Look at the history of this translation: https://hosted.weblate.org/translate/go-map/glossary/pt/?checksum=ea59ef52c0d6ec9c&sort_by=source#history

Every time you try to delete those 2 strings, it "deletes/clears" most of the Portuguese glossary, leaving the message "Found duplicated string" in the history.

The same thing happened to me, but I thought it was because I created those 2 entries that I tried to delete in the glossary.
